formfactory/admin.py

```diff
@@ -63,13 +62,30 @@ class WizardActionThroughInline(admin.StackedInline):
 class WizardAdmin(admin.ModelAdmin):
     form = forms.WizardAdminForm
     model = models.Wizard
-    list_display = ["title"]
-    inlines = [FormThroughInline, WizardActionThroughInline]
+    inlines = [WizardFormThroughInline, WizardActionThroughInline]
     prepopulated_fields = {"slug": ["title"]}
 
 
+class FieldGroupThroughInline(admin.StackedInline):
+    form = forms.FieldGroupThroughAdminForm
+    model = models.FieldGroupThrough
+
+
+class FormFieldGroupAdmin(admin.ModelAdmin):
```

Comment:
How about we add `prepopulated_fields = {"slug": ["title"]}` to this `ModelAdmin`?